src/Sarif.Multitool/RewriteCommand.cs

```diff
@@ -24,7 +24,9 @@ public int Run(RewriteOptions rewriteOptions)
             try
             {
                 rewriteOptions = ValidateOptions(rewriteOptions);
-                
+
+                DriverUtilities.VerifyOutputFileCanBeCreated(rewriteOptions.OutputFilePath, rewriteOptions.Force, _fileSystem);
```

Comment:
>VerifyOutputFileCanBeCreated [](start = 32, length = 28)

Seems like this should be in ValidateOptions?

Reply:
I'll try to put it there. The problem is that `ValidateOptions` returns a `bool` rather than throwing. I'll try to make `VerifyOutputFileCanBeCreated` fit into that pattern.

---
In reply to: [311712516](https://github.com/microsoft/sarif-sdk/pull/1625#discussion_r311712516) [](ancestors = 311712516)

Reply:
I did this for some commands. For others it was a bit awkward and IMO not worth the time in this PR.

---
In reply to: [311806351](https://github.com/microsoft/sarif-sdk/pull/1625#discussion_r311806351) [](ancestors = 311806351,311712516)